lib/src/validator.dart

```diff
@@ -159,6 +160,7 @@ abstract class Validator {
       PubspecTypoValidator(),
       LeakDetectionValidator(),
       SizeValidator(),
+      DevtoolsPluginValidator(),
```

Comment:
nit: rename to `DevToolsExtensionValidator` since that matches the `extension` convention.

Reply:
Done